calico_node/Makefile

```diff
@@ -318,9 +336,17 @@ st-checks:
 	# running on the host.
 	iptables-save | grep -q 'calico-st-allow-etcd' || iptables $(IPT_ALLOW_ETCD)
 
+calico-cni-plugin:
+	wget -O $@ https://github.com/projectcalico/cni-plugin/releases/download/v1.11.0/calico
```

Comment:
Not sure if this is intended long term but we probably want to pull the CNI version from versions.yml

Reply:
Yes, that makes sense. I will do that.